ui/src/user/Login.tsx

```diff
@@ -28,17 +28,21 @@ class Login extends Component<Stores<'currentUser'>> {
                         <form onSubmit={this.preventDefault} id="login-form">
                             <TextField
                                 autoFocus
+                                id="username"
                                 className="name"
                                 label="Username"
+                                name="username"
```

Comment:
Does name affects accessibility? For `id` it's documented in the mui link, but for name there is no additional notice for accessibility.

Reply:
Thanks for the explanation!

Reply:
You're welcome!